src/bin/main.rs

```diff
@@ -38,7 +41,7 @@ use yew_agent::Spawnable;
 // To load in the Fibonacci example, uncomment the CONTENT and fib_model lines
 // and comment the code, language, and text_model lines. IMPORTANT:
 // rename fib_model to text_model to have it work.
-const CONTENT: &str = include_str!("../../static/assembly_examples/fibonacci.asm");
+const CONTENT: &str = include_str!("../../static/assembly_examples/riscv_fib_recursive.asm");
```

Comment:
Since MIPS is still the default, can we switch this back to the original?

Reply:
May I also add, if it's possible to toggle between the two programs when the language is selected? And if you get that working, you guys might have an idea on creating drop-down menu to load in the examples programs. Just a suggestion.